interfaces/cython/cantera/composite.py

```diff
@@ -653,6 +664,60 @@ def write_csv(self, filename, cols=('extra','T','density','Y'),
             for row in data:
                 writer.writerow(row)
 
+    def to_pandas(self, cols=('extra', 'T', 'density', 'Y'),
+                  *args, **kwargs):
+        """
+        Returns the data specified by *cols* in a single Pandas DataFrame.
+
+        Additional arguments are passed on to `collect_data`. This method works
+        only with 1D SolutionArray objects and requires a working Pandas
+        installation. Use pip or conda to install pandas to enable this method.
+        """
+
+        # local import avoids explicit dependence of cantera on Pandas
+        import pandas as pd
+
+        data, labels = self.collect_data(cols, *args, **kwargs)
+        return pd.DataFrame(data=data, columns=labels)
+
+    def write_hdf(self, filename, cols=('extra', 'T', 'density', 'Y'),
+                  key='df', mode=None, append=None, complevel=None,
+                  *args, **kwargs):
+        """
+        Write data specified by *cols* to a HDF container file named *filename*.
+        Note that it is possible to write multiple data entries to a single HDF
+        container file, where *key* is used to differentiate data.
+
+        Internally, every HDF data entry is a `pandas.DataFrame` generated by
+        the `to_pandas` method.
+
+        :param filename: name of the HDF container file; typical file extensions
+            are `.hdf`, `.hdf5` or `.h5`.
+        :param cols: A list of any properties of the solution being exported.
+        :param key: Identifier for the group in the container file.
+        :param mode: Mode to open the file {None,'a','w','r+}.
+        :param append: use less efficient structure that makes HDF entries
+            appendable or append to existing appendable HDF entry {None,True,False}.
+        :param complevel: Specifies a compression level for data {None,0-9}.
+            A value of 0 disables compression.
+
+        Arguments *key*, *mode*, *append*, and *complevel* are mapped to
+        parameters for `pandas.DataFrame.to_hdf`; the choice `None` for *mode*,
+        *append*, and *complevel* results in default values set by Pandas.
+
+        Additional arguments (i.e. *args* and *kwargs*) are passed on to
+        `collect_data` via `to_pandas`; see `collect_data` for further
+        information. This method works only with 1D SolutionArray objects
+        and requires working installations of Pandas and PyTables. Use
+        pip or conda to install pandas and tables to enable this method.
```

Comment:
```suggestion
        pip or conda to install pandas and pytables to enable this method.
```

Reply:
pip installs PyTables as `pip install tables`, so the naming is not consistent, see [PyPI](https://pypi.org/project/tables/). Will leave this instance as is.

Reply:
But conda calls it `pytables`, so perhaps the last sentence of this should be "These packages can be installed using pip (`pandas` and `tables`) or conda (`pandas` and `pytables`)."

Reply:
No problem. I had originally cut & paste this from an error message raised by pandas, but your suggestion is much cleaner.